### ISSUE TITLE
[Masthead/Footer] Accessibility - The <label> element does not have descriptive text that identifies the expected input

### ISSUE THREAD

**Issue by abhin07** [carbon-design-system/carbon-for-ibm-dotcom#6891]
IBM Equal Access Accessibility Checker
The <label> element does not have descriptive text that identifies the expected input

Element showing the error
```
<label class="bx--label" part="label-text" for="input">
```

**Comment by RobertaJHahn** [comment]
@Praveen-sr Could you work with Ram to see whether this bug identified by the IBM Equal Access Accessibility Checker is valid? If it is, could you let us know what the priority would be. Thanks.

**Comment by kennylam** [comment]
This can be safely disregarded as this element is already hidden and will not interfere with assistive technologies.

cc: @Praveen-sr @RobertaJHahn
